### DISCUSSION TITLE
[iOS] complier error of google/protobuf

### DISCUSSION THREAD

**Discussion by EricDLee-sz** [trustwallet/wallet-core#1263] [Q&A] [answered]
**Describe the bug**
I have integrated wallet-core via Cocoapods but in Development Pods way. Any idea on this issue? Or is it possible to install wallet-core in this way? If the answer is NO, how can I use a forked repo?

**To Reproduce**
1. download repo
2. install pod using 'path'
3. build iOS project

**Expected behavior**
Project build successfully

**Screenshots**
<img width="1389" alt="image" src="https://user-images.githubusercontent.com/54877716/105973207-6ab5f480-60c7-11eb-8389-dd186c6a4204.png">

**Additional context**

**Comment by hewigovens** [comment]
try this way
```ruby
pod 'TrustWalletCore', git: 'https://github.com/trustwallet/wallet-core', branch: 'master'`
```

we have a pull request enables installing from local path: https://github.com/trustwallet/wallet-core/pull/1189, you still need to setup wallet core development environment, the best way is building and testing in `swift/TrustWalletCore.xcworkspace` first then integrate it to your app

**Reply by EricDLee-sz** [reply]
Sorry to bother, but plz treat me like 3 years old😛. So here is what I'm going to do:

1. clone `TrustWalletCore` to local, run `bootstrap.sh` to setup development environment
2. create a new project, add  `pod 'TrustWalletCore', path: '~/user/desktop/wallet-core`' to podfile then run `pod install`

But sadly is not working😞

Seems like there is no `WalletCore-development.podspec` I can use.

**Reply by hewigovens** [reply]
I said master branch doesn't support installing pod from local path, try podspec in pr 1189: https://github.com/trustwallet/wallet-core/blob/450604579fbc0ffa90d756d00b0f96d404f63f10/WalletCore.podspec

**Comment by hewigovens** [comment, marked as answer]
please check example on master: https://github.com/trustwallet/wallet-core/blob/master/swift/Example/Podfile
